### DISCUSSION TITLE
JSON.parse: unexpected character at line 2 column 5 of the JSON data

### DISCUSSION THREAD

**Discussion by Zhgong** [walinejs/waline#719] [Q&A] [answered]
系统版本：

* Hexo v5.4.0
* NexT v8.8.2
* Waline v1.5.

问题：
提交评论的时候会出现如截图中的报错信息。

![Screenshot from 2021-12-19 23-16-52](https://user-images.githubusercontent.com/7544688/146692766-189b14e0-687a-4eca-94a9-bb9bbda231e2.png)
2

**Comment by lizheming** [comment]
提供下地址我去看看，或者你看下浏览器 DevTool Network 看一下 `<serverURL>/comment` 接口的返回吧。

**Reply by Zhgong** [reply]
博客地址：https://justdong.net/2021/12/18/13hexo-next-waline/

我没有找到`<serverURL>/comment`的返回，请看下面的截图。你看看我哪里设置得不对。

![Screenshot from 2021-12-20 11-22-44](https://user-images.githubusercontent.com/7544688/146752444-a5114fdc-7d1e-4016-97ea-32ebad409c93.png)

![Screenshot from 2021-12-20 11-22-51](https://user-images.githubusercontent.com/7544688/146752459-b27b06ec-9222-40fc-9e10-7610c771a482.png)

**Comment by lizheming** [comment, marked as answer]
serverURL 配置有问题，改一下吧

![image](https://user-images.githubusercontent.com/424491/146762039-2d1ebbd5-fecb-49c3-a5ba-f88d037872ea.png)

**Reply by Zhgong** [reply]
多谢你的回复。按照你建议的方法问题解决了。
